common/stages/Functions/answerfile

```diff
@@ -37,6 +37,14 @@ read_xml_tag()
     sed -ne 's/^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\([^<]*\)<\/'"${TAG}"'>.*$/\2/p' <${FILE}
 }
 
+# This assumes the tag is on a single line in the file.
+sanitize_xml_tag()
+{
+    FILE="$1"
+    TAG="$2"
+    sed -i -e 's/\(^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\)\([^<]*\)\(<\/'"${TAG}"'>.*$\)/\1SANITIZED\4/' ${FILE}
```

Comment:
Good catch on the extra `\s*`.  I have it down to this:
`sed -i -e "s|\(<${TAG}\(\s\+[^>]*\)*>\)[^<]*\(</${TAG}>\)|\1SANITIZED\3|" ${FILE}`

Should I squash fixes into the original commits or push additional patches into the PR branch?